src/sink_http.c

```diff
@@ -216,7 +216,7 @@ static int serialize_metrics(struct http_sink* sink, metrics* m, void* data) {
 
     int post_len = 0;
     char* post_data = strbuf_get(post_buf, &post_len);
-    strbuf_free(post_buf, false);
+    strbuf_free(post_buf, true);
```

Comment:
Don't want this - we would free the char* array in the strbuf here, and pass a freed memory to the lifoq